charts/pega/charts/installer/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: installer
+version: 0.1.0
```

Comment:
`0.1.0`?  Should this be brought up to be in sync with the rest of the Pega chart?